form.py

```diff
@@ -0,0 +1,59 @@
+from flask_wtf import Form
+from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
+from flask_wtf.file import FileField
+from wtforms.validators import DataRequired, Email, URL
+from werkzeug import secure_filename
+
+class ApplicationForm(Form):
+    name = StringField('Name', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(),Email()])
+    phone = StringField('Phone')
+    city = StringField('City')
+    state = StringField('State')
+    zipcode = StringField('Zip Code')
+    citizenship = BooleanField('Verify you are a U.S. Citizen or Green Card holder')
+    primary_skillset = SelectField('Please indicate your primary skillset. Your response here will be used primarily in assigning reviewers for your application.', choices=[
+           ('acquisition_specialist', 'Acquisition Specialist'),
+           ('back_end_developer', 'Back End Developer'),
+           ('content_strategist', 'Content Strategist'),
+           ('content_writer', 'Content Writer'),
+           ('data_architect', 'Data Architect'),
+           ('devops_engineer', 'DevOps Engineer'),
+           ('front_end_developer', 'Front End Developer'),
+           ('full_stack_developer', 'Full Stack Developer'),
+           ('product_lead', 'Product Lead'),
+           ('project_or_program_manager', 'Project / Program Manager'),
+           ('ux_research_and_design', 'UX Research and Design'),
+           ('technical_architect', 'Technical Architect'),
+           ('technical_recruiter', 'Technical Recruiter'),
+           ('visual_designer', 'Visual Designer'),
+        ])
+    other_skills = TextAreaField('Other skills you have')
+    link1 = StringField('Link 1', validators=[URL()])
+    link2 = StringField('Link 2', validators=[URL()])
+    link3 = StringField('Link 3', validators=[URL()])
+    why_join_18f = TextAreaField('Why do you want to join 18F?')
+    resume = TextAreaField('Resume')
+    fulltime_ok = BooleanField('Verify your ability to work full-time')
+    location_pref = SelectField('Select your preferred work location', choices=[
+            ('chi', 'Chicago'),
+            ('ny', 'New York'),
+            ('sf', 'San Francisco'),
+            ('dc', 'Washington, DC'),
+            ('rmt', 'Remote')
+        ])
+    is_veteran = RadioField('Are you a Veteran of the U.S. Armed Forces or are you eligible for "derived" preference?',
+            choices=[('', 'No'), ('yes', 'Yes')])
```

Comment:
I was going by the default falsey values mentioned here as being ('',
'false')
http://wtforms.readthedocs.org/en/latest/fields.html?highlight=false#wtforms.fields.BooleanField

On Thu, Sep 24, 2015 at 4:22 PM, Fureigh <notifications@github.com> wrote:

> In form.py
> <https://github.com/18F/joining-18f-app/pull/1#discussion_r40367680>:
>
> > +    other_skills = TextAreaField('Other skills you have')
> > +    link1 = StringField('Link 1', validators=[URL()])
> > +    link2 = StringField('Link 2', validators=[URL()])
> > +    link3 = StringField('Link 3', validators=[URL()])
> > +    why_join_18f = TextAreaField('Why do you want to join 18F?')
> > +    resume = TextAreaField('Resume')
> > +    fulltime_ok = BooleanField('Verify your ability to work full-time')
> > +    location_pref = SelectField('Select your preferred work location', choices=[
> > +            ('chi', 'Chicago'),
> > +            ('ny', 'New York'),
> > +            ('sf', 'San Francisco'),
> > +            ('dc', 'Washington, DC'),
> > +            ('rmt', 'Remote')
> > +        ])
> > +    is_veteran = RadioField('Are you a Veteran of the U.S. Armed Forces or are you eligible for "derived" preference?',
> > +            choices=[('', 'No'), ('yes', 'Yes')])
>
> Should this say ('no', 'No') instead of ('', 'No')?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/joining-18f-app/pull/1/files#r40367680>.
>



-- 
Alison Rowland
Technical Lead, https://fbopen.gsa.gov | 18F
202-317-0124